src/Humanizer/CollectionHumanizeExtensions.cs

```diff
@@ -15,46 +15,46 @@ public static class CollectionHumanizeExtensions
         /// using the default separator for the current culture.
         /// </summary>
         /// <returns></returns>
-        public static string Humanize<T>(this IEnumerable<T> collection)
+        public static string Humanize<T>(this IEnumerable<T> collection, StringJoinOptions options = StringJoinOptions.Default)
```

Comment:
@hazzik  Can you add more information around this? I think the code will work the same (but I might be wrong). Could it break something? What am I missing?